src/utils/actions.ts

```diff
@@ -1,7 +1,7 @@
 import { isFSA, FluxStandardAction } from './isFSA'
 
 // Certain actions that we should never replay across stores
-const blacklist = [/^@@/, /^redux-form/]
+const defaultDenyList = [/^@@/, /^redux-form/]
```

Comment:
Should we actually default the options in the `RendererStateSyncEnhancerOptions` / `MainStateSyncEnhancerOptions` I know it will be duplicated, but it would make it also more transparent to the user what actions get filtered out by default, rather than having this baked into the library as a _magic_ thing. Thoughts?

Reply:
The main question I thought about is if the user should be able to override the defaults with their own denyList. Otherwise I would just document it in the README.

Reply:
Overriding the defaults seem reasonable as long as we document the default values.

Reply:
What I mean is that by default we should keep these values, but a user should be able to replace them if they provide an array as argument, so current implementation looks good to me. Though we should add an example of how to use this without replacing them altogether.

Reply:
I thought about this topic again and I think we should export the default denyList so that users can still append it or introduce another option to not override the default list. WDYT?

Reply:
it makes sense :)

Reply:
Is there one solution you would prefer?